htdocs/knowledgemanagement/knowledgerecord_card.php

```diff
@@ -185,6 +192,13 @@
 	// Common attributes
 	include DOL_DOCUMENT_ROOT.'/core/tpl/commonfields_add.tpl.php';
 
+	//Language of question/response
```

Comment:
Can you include this into the commonfields_add.tpl.php so every module that use a 'lang' field (name reserved) will benefit the language selector/beautifier.